WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/EditorMedia.kt

```diff
@@ -298,6 +298,10 @@ class EditorMedia @Inject constructor(
         listener.onMediaUploadFailed(media.id.toString(), error.type.name)
     }
 
+    fun onMediaUploadPaused(listener: EditorMediaUploadListener, media: MediaModel) = launch {
+        listener.onMediaUploadPaused(media.id.toString())
+    }
```

Comment:
I imagine we should still track an event of `EDITOR_UPLOAD_MEDIA_FAILED` for paused media, or implement a new event. WDYT?